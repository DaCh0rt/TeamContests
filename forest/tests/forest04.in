10 148
-10000 -10000
-10000 10000
10000 10000
10000 -10000
4 4
5 5
0 0
3 3
1 1
2 2
