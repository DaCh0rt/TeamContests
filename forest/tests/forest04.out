80929.91 411908813.45
